cdap-standalone/bin/cdap.bat

```diff
@@ -173,11 +189,12 @@ set class=co.cask.cdap.data2.transaction.TransactionManagerDebuggerMain
 REM Skip first parameter
 for /f "usebackq tokens=1*" %%i in (`echo %*`) DO @ set params=%%j
 
-%JAVACMD% %DEFAULT_JVM_OPTS% %HADOOP_HOME_OPTS% %TOKEN_FILE_OPTS% -classpath %CLASSPATH% %class% %params%
+"%JAVACMD%" %DEFAULT_JVM_OPTS% %HADOOP_HOME_OPTS% %TOKEN_FILE_OPTS% -classpath "%CLASSPATH%" %class% %params%

```

Comment:
No, because it is already quoted. Using a `CDAP_HOME` with a space in it worked fine:
`SET HADOOP_HOME_OPTS=-Dhadoop.home.dir="%CDAP_HOME%\libexec"`